addons/project/models/project_task.py

```diff
@@ -1338,6 +1336,28 @@ def _task_message_auto_subscribe_notify(self, users_per_task):
                     model_description=task_model_description,
                     mail_auto_delete=False,
                 )
+            unassigned_user_name = ', '.join(unassigned_users.mapped('name'))
+            if unassigned_user_name:
+                subtype_id_unassigned = self.env['ir.model.data']._xmlid_to_res_id('project.mt_task_unassigned', raise_if_not_found=False)
+                followers_with_subtype = task.message_follower_ids.filtered(lambda follower: subtype_id_unassigned in follower.subtype_ids.ids)
+                if followers_with_subtype:
+                    values = {
+                        'object': task,
+                        'model_description': task.env['ir.model']._get(task._name).display_name,
+                        'access_link': task._notify_get_action_link('view'),
+                        'unassigned_user_name': unassigned_user_name,
+                    }
+                    unassignation_msg = self.env['ir.qweb']._render('project.project_message_user_unassigned', values, minimal_qcontext=True)
+                    unassignation_msg = self.env['mail.render.mixin']._replace_local_links(unassignation_msg)
+                    task.message_notify(
+                        subject=_("%(unassignee_name)s has been unassigned to %(task_name)s", unassignee_name=unassigned_user_name, task_name=task.display_name),
+                        body=unassignation_msg,
+                        partner_ids=followers_with_subtype.partner_id.ids,
+                        record_name=task.display_name,
+                        email_layout_xmlid='mail.mail_notification_layout',
+                        model_description=task.env['ir.model']._get(task._name).display_name,
+                        mail_auto_delete=False,
+                    )
```

Comment:
I would put this in a different method, it's a bit messy having everything in one